dependency_management/compare_fedora_versions

```diff
@@ -52,11 +52,15 @@ def main():
         "'rawhide' or 'f<n>' where <n> is some natural number"
     )
     parser.add_argument("--release", action="store", default="rawhide", help=help_text)
+    help_text = "Manifest path for the cargo metadata command"
+    parser.add_argument(
+        "--manifest-path", action="store", dest="manifestpath", help=help_text
```

Comment:
It's conventional to put an underscore in the dest value if there is a dash in the option, as ```manifest_path```.